README.md

```diff
@@ -66,3 +66,56 @@ sensor:
         friendly_name: "Eilles Frankfurt Price"
         value_template: "{{ state_attr('sensor.toogoodtogo_eilles_frankfurt', 'price') }}"
 ```
+
+## Get a list of all toogoodtogo sensors:
+
+Add the following piece of code into /developer-tools/template in HomeAssistant. (Remove the last comma)
+
+```yaml
+{%- for state in states -%}
+    {%- if (state.entity_id.startswith('sensor.toogoodtogo_'))-%}
+      {{state.entity_id}},
+    {%- endif -%}
+{%- endfor -%}
+```
+
+## Example HomeAssistant Automation
+
+```yaml
+alias: TooGoodToGo Notification
+description: "Sends a notification when a toogoodtogo offer becomes available"
+trigger:
+  - platform: state
+    entity_id:
+      >- # This is your list of toogoodoto sensors which are generated (Copy paste the list from above)
+      sensor.toogoodtogo_1,sensor.toogoodtogo_2,sensor.toogoodtogo_3
+    attribute: stock_available
+    from: false
+    to: "true"
```

Comment:
Just out of curiosity, why is this one time a string, and one time a boolean?

Reply:
Yeh I was checking this out myself aswell, but I think ha has some troubles with booleans in templates. Since it keeps trying to transform the Boolean back into strings each time I go into the automation edit.

Reply:
I will edit them both to booleans shortly, also have an 'unnotify' automation which removes an notification on expiry.